.github/dependabot.yml

```diff
@@ -6,8 +6,8 @@ updates:
     directory: "/frontend"
     schedule:
       interval: "daily"
-    # Allow one update PR at a time:
-    open-pull-requests-limit: 1
+    # Allow a limited number of update PRs at a time:
```

Comment:
Perhaps a good idea to add to the comment why we want to have a limited number of update PRs (time, cost, overview, ...)

Reply:
Added a bit more. I think its mostly about keeping the number of PRs manageable and slowly introducing automated dependency updates without creating too much chaos. I can see us fully removing this restriction at some point.